final_task/README.md

```diff
@@ -1,3 +1,56 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+RSS_READER
+RSS reader is a command-line utility.
+
+Usage
+usage: rss_reader.py [-h] [--source SOURCE] [--version] [--json] [--verbose]
+[--limit LIMIT] [--date DATE]
+
+Pure Python command-line RSS reader.
+
+optional arguments:
+
+-h, --help            show this help message and exit
+--source SOURCE            RSS URL
+--version            Print version info
+--json            Print result as JSON in stdout
+--verbose            Outputs verbose status messages
+--limit LIMIT            Limit news topics if this parameter provided
+--date DATE            News from the specified day will be printed out. Format: YYYYMMDD
+It is mandatory to specify date or/and time.
+If both are specified, then news will be searched by date and by source.
+
+Json structure
+[
+    {
+            "feed": [feed],
+            "title": [title],
+            "date": [date],
+            "link": [link],
+            "text": [text],
+            "image links": [
+                [link1]
+                [link2]
+                ...
+            ]
+    },
+        ...
+]
+
+Local storage
+All read news is saved in database by using Mysql. You should have database final_task_database with table news_cache
+new-cache structure:
+    feed:longtext
+    title:longtext
+    date:date
+    link:longtext
+    image_description:longtext
+    new_description:longtext
+    image_links:longtext
```

Comment:
Are you sure this is correct table structure?

I have set everything up, but I get warnings that date format is invalid and exception when trying to write news to the database:

![los](https://user-images.githubusercontent.com/43743521/69367722-d887b480-0ca9-11ea-8607-6b4926a1e89c.png)

Reply:
Sorry, my mistake. Format of date must be Datetime.

Reply:
Sorry again, date must have format date. I found mistake in write_news_to_database function, now I am fixing it

Reply:
Please fix it, and test it. I will look your pr once again  you will push changesa.